trees/lib/models/binarySearchTree.js

```diff
@@ -67,6 +74,23 @@ class BinarySearchTree {
     }
   }
 
+  /***********************************
+  *    FIND MAX NODE ANY BIN TREE    *
+  ************************************/
+  static findMaxBinaryTree(tree) {

```

Comment:
There are some naming issues with this method.
Are you finding the maximum binary tree, or are you finding a maximum node, or are you finding a maximum value?
In this case, it compares and returns a `value` (which is correct), so `findMaxValue()` would be a much better name.